TLM/TLM/UI/SubTools/SpeedLimits/SpeedLimitsTool.cs

```diff
@@ -142,6 +143,8 @@ public SpeedLimitsTool(TrafficManagerTool mainTool)
 
         public override void OnToolGUI(Event e) {
             base.OnToolGUI(e);
+            var oldMatrix = GUI.matrix;
+            GUI.matrix = UIScaler.ScaleMatrix;
```

Comment:
what is this matrix switching thing?
currently on the train, can't look it up easily.
is GUI.matrix our code? 
if not and we can't change it than this is ok. 
if its hours we really should change it :)

Reply:
`GUI.matrix` is Unity code for immediate gui, works pretty much like `GUI.color`, but obviously changes different thing 😉

Reply:
kk .. in that case the code is fine.